go/gin/go.sum

```diff
@@ -0,0 +1,38 @@
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
```

Comment:
How do you get this file ? By a `go get` ?

Reply:
Just try the `go mod tidy`